rpc-interface/src/types.rs

```diff
@@ -555,6 +564,7 @@ pub struct Transaction {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub confirmations: Option<u32>,
     pub size: usize,
+    pub related_addresses: Vec<Address>,
```

Comment:
```suggestion
    pub related_addresses: BTreeSet<Address>,
```
Duplicates aren't meaningful, I think.

Reply:
The method that computes them uses a `HashSet` already. But yeah, it could be used here, too. I didn't check if it still outputs as an array in JSON.

Reply:
updated :+1: